boards/arm/mec172xevb_assy6906/debug/power_debug.c

```diff
@@ -0,0 +1,15 @@
+#include <zephyr/device.h>
+#include <zephyr/drivers/led.h>
+#include <soc/arm/microchip_mec/mec172x/soc_power_debug.h>
+
+const struct device *led = DEVICE_DT_GET(DT_ALIAS(led0));
+
+void pm_debug_function(uint32_t debug)
```

Comment:
@jjdalynh the original code intended to help track SoC sleep state entry to debug anomalous entry/exit or check latency issues, the goal was to perform a quick operation by accessing registers directly

1) By replacing as led driver call this won't be as short as before
2) It may block deep sleep states entry since led_on call may prevent main clock to be turned off
cc @scottwcpg